subprojects/docs/src/main/resources/header.html

```diff
@@ -165,50 +164,98 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
+                <ul id="learning-the-basics-dependency-management">
+                    <li><a href="core_dependency_management.html">What is Dependency Management?</a></li>
+                    <li><a href="dependency_management_terminology.html">Terminology</a></li>
+                    <li><a href="declaring_repositories.html">Declaring Repositories</a></li>
+                    <li><a href="declaring_dependencies.html">Declaring Dependencies</a></li>
+                    <li><a href="library_vs_application.html">Developing Libraries</a></li>
+                    <li><a href="viewing_debugging_dependencies.html">Debugging Dependencies</a></li>
+                    <li><a href="dependency_resolution.html">Dependency Resolution</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#advanced-techniques" aria-expanded="false" aria-controls="advanced-techniques">Advanced Techniques</a>
-                <ul id="advanced-techniques">
-                    <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing Parallel Tasks</a></li>
-                    <li><a href="test_kit.html">Testing Your Build with TestKit</a></li>
-                    <li><a href="ant.html">Using Ant from Gradle</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#declaring-dependency-versions" aria-expanded="false" aria-controls="declaring-dependency-versions">Choosing Versions</a>
+                <ul id="declaring-dependency-versions">
+                    <li><a href="single_versions.html">Declaring a Simple Version</a></li>
+                    <li><a href="rich_versions.html">Declaring a Rich Version</a></li>
+                    <li><a href="dynamic_versions.html">Handling Changing Versions</a></li>
+                    <li><a href="dependency_locking.html">Locking Versions</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
-                <ul id="sample-gradle-builds">
-                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
-                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#controlling-transitive-dependencies" aria-expanded="false" aria-controls="controlling-transitive-dependencies">Controlling Transitives</a>
+                <ul id="controlling-transitive-dependencies">
+                    <li><a href="dependency_constraints.html">Declaring Constraints</a></li>
```

Comment:
This section contains the part about overriding and excluding. I wonder if we should not promote at least the first one level up. After all this is a really common issue and deserves an easier to find section.
For the later, it could be a subsection of the override part and not move one level up indeed.

Reply:
Renamed and split up as discussed